docs/api/rundeck-api.md

```diff
@@ -1069,24 +1069,323 @@ Response:
 
     204 No Content
     
-## Config Refresh ##
 
-Make the Rundeck server re-read the config properties file.
+## Config Management ###
+Manage your plugin and custom configuration properties via API.
+
+### Create or Update Configurations ####
+    POST api/36/config/save
+Create or update configs and properties. POST fails if any configs are invalid.
 
 **Request:**
 
+Headers
+* `Content-Type` Required
+  * Accepts: `application/json`
+
+Body
+* `key` Required
+  * Represents either a new config to be created, or an existing config to be updated.
+  * Accepts: Any `string`
+* `value` Required
+  * The value for the desired config `key`.
+  * Accepts: Any `string`
+* `strata` Optional
+  * Whether the config should apply to the current server (`Server`), or to all servers in the cluster (`default`).
+  * Accepts: `Server`, `default`
+  * Default: `default`
+
+Example JSON
+```json
+[
+  {
+    "key": "myCustomConfig",
+    "value": "newValueForCustomConfig",
+    "strata": "default"
+  },
+  {
+    "key": "myNewCustomConfig",
+    "value": "valueOfNewCustomConfig",
+    "strata": "default"
+  }
+]
+```
+
+**Responses**
+
+    200 OK
+
+All configs were successfully saved or updated. A payload reflecting save or creation status is returned.
+
+Headers  
+`Content-Type: application/json`:
+
```

Comment:
do you want to add in the other use cases as mentioned in the work request?

{
    "msg": "Saved, with some invalid configs ignored",
    "created": [],
    "updated": [
        "r.feature.notificationsOwnThread.enabled"
    ],
    "ignored": [
        "rundeck.feature.notificationsOwnThread.enabled"
    ]
}

Reply:
I changed that in the ticket because there was some slack discussion that we wanted the whole thing to fail if any data was invalid